packages/design-system-react/src/components/AvatarNetwork/AvatarNetwork.test.tsx

```diff
@@ -65,7 +66,7 @@ describe('AvatarNetwork', () => {
         <AvatarNetwork
           name="ACME"
           src={src}
-          imageProps={{ 'data-testid': 'img', id: 'img-id' } as any}
+          imageProps={{ 'data-testid': 'img', id: 'img-id' }}
```

Comment:
Same fixes for all imageProps types see this comment https://github.com/MetaMask/metamask-design-system/pull/674/files#r2094696645